cng/aes.go

```diff
@@ -62,16 +67,20 @@ func (c *aesCipher) Encrypt(dst, src []byte) {
 }
 
 func (c *aesCipher) Decrypt(dst, src []byte) {
-	if subtle.InexactOverlap(dst, src) {
-		panic("crypto/cipher: invalid buffer overlap")
-	}
 	if len(src) < aesBlockSize {
 		panic("crypto/aes: input not full block")
 	}
 	if len(dst) < aesBlockSize {
 		panic("crypto/aes: output not full block")
 	}
 
+	// cypher.Block.Decrypt() is documented to decrypt one full block
```

Comment:
Is this a duplicate code block that could be extracted?

Reply:
Yes they can 😄. Done!